README.md

```diff
@@ -1,3 +1,33 @@
-# DevOps
+# Helpful Engineers GitHub Organization
```

Comment:
The title should be changed to «DevOps team» or something along these lines.
Preferably, including a link to the Slack channel.

Reply:
👍